packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -116,6 +116,11 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
    */
   menuIconProps?: IIconProps;
 
+  /**
+   * The title to announce the dropdown button if this button is split.
```

Comment:
Accessible label for cheveron icon if this button is split

Reply:
Changed.